dev-utils/update-4x-branch.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Create a PR to update the 4.x branch to match the state of "main" for the
+# just-tagged release. The 4.x branch needs to be updated for the current docs
+# build.
+#
+
+if [ "$TRACE" != "" ]; then
+    export PS4='${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+    set -o xtrace
+fi
+set -o errexit
+set -o pipefail
+
+function fatal {
+    echo "$(basename $0): error: $*"
+    exit 1
+}
+
+TOP=$(cd $(dirname $0)/../ >/dev/null; pwd)
+WRKDIR=${TOP}/build/update-4x-branch
+
+echo "# Creating working git clone in: ${WRKDIR}/apm-agent-nodejs"
+rm -rf $WRKDIR
+mkdir -p $WRKDIR
+cd $WRKDIR
+git clone git@github.com:elastic/apm-agent-nodejs.git
+cd apm-agent-nodejs
+
+TARGTAG=$(git tag --points-at HEAD)
+if [[ ! ("$TARGTAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]$) ]]; then
+    fatal "the tag on HEAD, '${TARGTAG}', does not look like a release tag"
+fi
+# echo "TARGTAG=$TARGTAG"
+
+LASTTAG=$(
+    git log --pretty=format:%h -30 | tail -n +2 | while read sha; do
+        possible=$(git tag --points-at $sha)
+        if [[ "$possible" =~ ^v[0-9]+\.[0-9]+\.[0-9]$ ]]; then
+            echo $possible
+            break
+        fi
+    done
+)
+if [[ -z "$LASTTAG" ]]; then
+    fatal "could not find previous release tag in last 30 commits"
```

Comment:
does this means we have a limit of commits between releases? I'm okay with that but then there should be a reminder somewhere

Reply:
I can bump it. 30 *does* seem a little low. This was just meant as a sanity guard.

Reply:
commit 41216321